sdk/ml/azure-ai-ml/azure/ai/ml/entities/_compute/aml_compute.py

```diff
@@ -124,7 +122,7 @@ def __init__(
         idle_time_before_scale_down: Optional[int] = None,
         identity: Optional[IdentityConfiguration] = None,
         tier: Optional[str] = None,
-        enable_node_public_ip: Optional[bool] = True,
+        enable_node_public_ip: bool = True,
```

Comment:
Why is Optional removed in this case?

Reply:
No need of optional keyword when default value is being passed in.